cx_Freeze/hooks/__init__.py

```diff
@@ -94,7 +94,10 @@ def load_clr(finder: ModuleFinder, module: Module) -> None:
     """The pythonnet package (imported as 'clr') needs Python.Runtime.dll
     in runtime."""
     dll_name = "Python.Runtime.dll"
-    finder.include_files(module.file.parent / dll_name, Path("lib", dll_name))
+    finder.include_files(
```

Comment:
We need to keep supporting the previous version, so you should check if the file exists in this new path or in the old one.